examples/mobile/UIComponents/components/search/search-bar.js

```diff
@@ -0,0 +1,25 @@
+(function (document, tau) {
+	var page = document.getElementById("search-bar-page"),
+		search;
+
+	function onSearchFocus() {
+		var popup = tau.widget.Popup(page.querySelector(".ui-popup"));
+
+		popup.open();
+	}
+
+	function onPageShow() {
+		search = page.querySelector("header .ui-appbar-title-container input[type='search']");
+		search.addEventListener("focus", onSearchFocus);
+	}
+
+	function onPageHide(ev) {
+		var page = ev.target;
+
+		page.removeEventListener("pageshow", onPageShow);
```

Comment:
How about adding the removeEventListener for the focus event.

Reply:
You are right. I'll fix it.

Reply:
Done.